pkgs/by-name/st/stackql/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, testers
+, stackql
+}:
+
+buildGoModule rec {
+  pname = "stackql";
+  version = "0.5.643";
+
+  src = fetchFromGitHub {
+    owner = "stackql";
+    repo = "stackql";
+    rev = "v${version}";
+    hash = "sha256-9W6bEI+5Q0Kgbd14sWKde3I6WIVz1ZxsXmR009mOPog=";
+  };
+
+  vendorHash = "sha256-xcly4jtdUkx/s+YWYFBVeuI2kQBu2oqbLN9ZKkHppkA=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/stackql/stackql/internal/stackql/cmd.BuildMajorVersion=0"
+    "-X github.com/stackql/stackql/internal/stackql/cmd.BuildMinorVersion=5"
+    "-X github.com/stackql/stackql/internal/stackql/cmd.BuildPatchVersion=643"
+    "-X github.com/stackql/stackql/internal/stackql/cmd.BuildCommitSHA=${src.rev}"
+    "-X github.com/stackql/stackql/internal/stackql/cmd.BuildShortCommitSHA=${src.rev}"
```

Comment:
These will not point to the hash, but instead to your version number

If you want the actual hash, you can use `fetchFromGitHub` with `leaveDotGit = true`, and parse the git log - alternatively, you can specify the hash for the version in `rev` instead.

If these aren't super important, you may also just leave them blank - many packages don't try to keep the git hashes in the builds, even if the underlying program supports it.

Reply:
removed as you've suggested. The only difference it appears to make is the version number has empty brackets ie.,
`stackql v0.5.643  ()`, which I think is ok.